tests/WE2E/machine_suites/coverage.derecho

```diff
@@ -0,0 +1,10 @@
+community
```

Comment:
Sorry for not catching this earlier...this should be the same list as coverage.cheyenne, no? Since EPIC is not testing on Cheyenne anymore I thought the idea was to move those tests to Derecho.

Reply:
@mkavulich - done, thank you for the comment